nbconvert/filters/tests/test_markdown.py

```diff
@@ -137,6 +137,18 @@ def test_markdown2html_math(self):
             "$$a<b&b<lt$$",
             "$$a<b&lt;b>a;a-b<0$$",
             "$$<k'>$$",
+            """$$x
+=
+2$$""",
+            r"""$$
+ b_{l_1,l_2,l_3}^{\phi\phi\omega} = -8\, l_1\times l_2 \,l_1\cdot l_2 \int_0^{\chi_*} d\chi \frac{W(\chi,\chi_*)^2}{\chi^2} \int_0^{\chi}d\chi' \frac{W(\chi',\chi)W(\chi',\chi_*)}{{\chi'}^2} \left[
+ P_\psi\left(\frac{l_1}{\chi},z(\chi)\right) P_\Psi\left(\frac{l_2}{\chi'},z(\chi')\right)
+- (l_1\leftrightarrow l_2)
+\right]
+$$""",
+            r"""\begin{equation*}
```

Comment:
What does this add over first example?

https://github.com/jupyter/nbconvert/blob/3e203ce4c5989192908720e646f3c41cd2dea038/nbconvert/filters/tests/test_markdown.py#L124-L128

Reply:
It doesn't need to be a raw string so please use a standard literal string and escape anything that needs to be escaped.